src/lib/gssapi/krb5/gssapi_krb5.h

```diff
@@ -96,6 +96,15 @@ GSS_DLLIMP extern const gss_OID_desc krb5_gss_oid_array[];
  */
 GSS_DLLIMP extern const gss_OID GSS_KRB5_CRED_NO_CI_FLAGS_X;
 
+/*
+ * This OID can be used with gss_inquire_cred_by_oid(0 to retrieve the
+ * impersonator name (if any).
+ *
+ * iso(1) member-body(2) United States(840) mit(113554) infosys(1) gssapi(2)
+ * krb5(2) krb5_gssapi_ext(5) get_cred_impersonator(14)
+ */
+GSS_DLLIMP extern const gss_OID GSS_KRB5_GET_CRED_IMPERSONATOR;
```

Comment:
@simo5 it's best to get approval from the responsible party for an OID arc to use an OID from that arc.  I don't know who is responsible for that arc in particular; I suspect I will need to dig through my email archives to be sure.  It's also not valid to use underscores in OID symbolic identifiers, so it probably wasn't me who assigned that.  I'm looking at the file that's supposed to be authoritative for that arc and the closest assignment in the 1.2.840.113554.1.2.2 arc is 1.2.840.113554.1.2.2.3 (which is for the user-to-user gss-krb5 mech).  It seems that someone hasn't been updating the file like they should.  (I don't have access to, but I in theory can get the people who do have access to do it for me.)